### PR TITLE
UI Tests Nested Buttons and Radio button pressed

### PR DESCRIPTION
**Pull request by manpreetkaurnventive**
GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?

- Other... Please describe: UI Tests

## What is the current behavior?

No UI Tests

## What is the new behavior?

Added new UI tests for nested buttons and textbox before changing text
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](https://github.com/unoplatform/uno/blob/master/README.md#uno-features)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [X] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [X] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)
- [X] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):
https://nventive.visualstudio.com/Umbrella/_sprints/taskboard/Umbrella%20Team/Umbrella/2020/Sprint%203?workitem=173212

### CODE REVIEW

File: src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ButtonTests/UnoSamples_Tests.Button_IsEnabled_Automated.cs [hunk 1]
```diff
@@ -256,5 +256,35 @@ void Check(bool expected, string msg)
...
+			var inner = _app.Marked("Inner");
+			var validateCommandTextBlock = _app.Marked("ValidateCommandTextBlock");
+
+			// Initial validation
+			Assert.AreEqual("", validateCommandTextBlock.GetDependencyPropertyValue("Text")?.ToString());
+
+			// Click on outer button and validate
+			_app.TapCoordinates(outer.FirstResult().Rect.X + 30, outer.FirstResult().Rect.Y + 30);
+			string command1 = validateCommandTextBlock.GetDependencyPropertyValue("Text")?.ToString();
+			if (!command1.Contains("OuterCommand"))
```
Comment (carldebilly) [inline src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ButtonTests/UnoSamples_Tests.Button_IsEnabled_Automated.cs]: Codacy Issue found: [Change this call to 'command1.Contains' to an overload that accepts a 'StringComparison' as a parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4958644)

Comment (carldebilly) [inline src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ButtonTests/UnoSamples_Tests.Button_IsEnabled_Automated.cs]: Codacy Issue found: ['command1' is null on at least one execution path.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4958644)

File: src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ButtonTests/UnoSamples_Tests.Button_IsEnabled_Automated.cs [hunk 2]
```diff
@@ -256,5 +256,35 @@ void Check(bool expected, string msg)
...
+			{
+				Assert.Fail("Outer button is not clicked");
+			}
+
+			// Click on inner button and validate
+			_app.TapCoordinates(inner.FirstResult().Rect.X + 30, inner.FirstResult().Rect.Y + 30);
+			string command2 = validateCommandTextBlock.GetDependencyPropertyValue("Text")?.ToString();
+			if (!command2.Contains("InnerCommand"))
```
Comment (carldebilly) [inline src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ButtonTests/UnoSamples_Tests.Button_IsEnabled_Automated.cs]: Codacy Issue found: ['command2' is null on at least one execution path.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4958644)

Comment (carldebilly) [inline src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ButtonTests/UnoSamples_Tests.Button_IsEnabled_Automated.cs]: Codacy Issue found: [Change this call to 'command2.Contains' to an overload that accepts a 'StringComparison' as a parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4958644)

File: src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ButtonTests/UnoSamples_Tests.Button_IsEnabled_Automated.cs (hunk 1 above)
Comment (agneszitte-nventive) [reply src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ButtonTests/UnoSamples_Tests.Button_IsEnabled_Automated.cs]: @manpreetkaurnventive 
```if (!command1.Contains("OuterCommand", StringComparison.InvariantCultureIgnoreCase))```

Comment (agneszitte-nventive) [reply src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ButtonTests/UnoSamples_Tests.Button_IsEnabled_Automated.cs]: @manpreetkaurnventive 
```if (command1 != null && !command1.Contains("OuterCommand", StringComparison.InvariantCultureIgnoreCase))```

File: src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ButtonTests/UnoSamples_Tests.Button_IsEnabled_Automated.cs (hunk 2 above)
Comment (agneszitte-nventive) [reply src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ButtonTests/UnoSamples_Tests.Button_IsEnabled_Automated.cs]: @manpreetkaurnventive 
```if (command2 != null && !command2.Contains("InnerCommand", StringComparison.InvariantCultureIgnoreCase))```

Comment (agneszitte-nventive) [reply src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ButtonTests/UnoSamples_Tests.Button_IsEnabled_Automated.cs]: @manpreetkaurnventive 
```if (command2 != null && !command2.Contains("InnerCommand", StringComparison.InvariantCultureIgnoreCase))```

File: src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ButtonTests/UnoSamples_Tests.Button_IsEnabled_Automated.cs [hunk 3]
```diff
@@ -256,5 +256,35 @@ void Check(bool expected, string msg)
...
+			var inner = _app.Marked("Inner");
+			var validateCommandTextBlock = _app.Marked("ValidateCommandTextBlock");
+
+			// Initial validation
+			Assert.AreEqual("", validateCommandTextBlock.GetDependencyPropertyValue("Text")?.ToString());
+
+			// Click on outer button and validate
+			_app.TapCoordinates(outer.FirstResult().Rect.X + 30, outer.FirstResult().Rect.Y + 30);
+			string command1 = validateCommandTextBlock.GetDependencyPropertyValue("Text")?.ToString();
+			if (command1 != null && !command1.Contains("OuterCommand"))
```
Comment (carldebilly) [inline src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ButtonTests/UnoSamples_Tests.Button_IsEnabled_Automated.cs]: Codacy Issue found: [Change this call to 'command1.Contains' to an overload that accepts a 'StringComparison' as a parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4958644)

File: src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ButtonTests/UnoSamples_Tests.Button_IsEnabled_Automated.cs [hunk 4]
```diff
@@ -256,5 +256,35 @@ void Check(bool expected, string msg)
...
+			{
+				Assert.Fail("Outer button is not clicked");
+			}
+
+			// Click on inner button and validate
+			_app.TapCoordinates(inner.FirstResult().Rect.X + 30, inner.FirstResult().Rect.Y + 30);
+			string command2 = validateCommandTextBlock.GetDependencyPropertyValue("Text")?.ToString();
+			if (command2 != null && !command2.Contains("InnerCommand"))
```
Comment (carldebilly) [inline src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ButtonTests/UnoSamples_Tests.Button_IsEnabled_Automated.cs]: Codacy Issue found: [Change this call to 'command2.Contains' to an overload that accepts a 'StringComparison' as a parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4958644)

**Pull request merged**